canonical-websites/docs.snapcraft.io
# 404 for topics not in category

- Set the category to category 15 ("doc")
- When requesting a topic, check its category_id is 15
- If not, raise a 404

Fixes https://github.com/ubuntudesign/base-squad/issues/329
Fixes https://github.com/canonical-websites/docs.snapcraft.io/issues/114

QA
--

`./run`

Check the site works.

Try to view a topic not in the doc category, e.g.:

http://0.0.0.0:8030/classic-confinement-request-dlang/9391

Check you see a 404.
